modules/extensions-support/src/test/java/org/mule/module/extension/internal/metadata/resolver/ContentResolverWithKeyResolver.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.metadata.resolver;
+
+import org.mule.api.metadata.MetadataKey;
+import org.mule.extension.api.metadata.MetadataContext;
+import org.mule.extension.api.metadata.MetadataResolver;
+import org.mule.extension.api.metadata.MetadataResolvingException;
+import org.mule.metadata.api.model.MetadataType;
+import org.mule.module.extension.internal.metadata.MetadataService;
+
+import java.util.List;
+
+public class ContentResolverWithKeyResolver implements MetadataResolver
```

Comment:
same here, this is the full path:
4.x-ce/modules/extensions-support/src/test/java/org/mule/module/extension/internal/metadata/resolver/ContentAndOutputResolverWithoutKeyResolver.java